filebeat/datadog_checks/filebeat/filebeat.py

```diff
@@ -210,11 +211,16 @@ def _compile_regexes(self):
 
 
 class FilebeatCheck(AgentCheck):
+
+    METRIC_PREFIX = "filebeat."
+
     def __init__(self, *args, **kwargs):
         AgentCheck.__init__(self, *args, **kwargs)
         self.instance_cache = {}
 
     def check(self, instance):
+        normalize_metrics = instance.get("normalize_metrics")
```

Comment:
Can you do:
```suggestion
        normalize_metrics = is_affirmative(instance.get("normalize_metrics"), False)
```

Reply:
Oops I forgot it needs to be imported- I think it's `from datadog_checks.base import is_affirmative`?